package.json

```diff
@@ -82,4 +82,4 @@
       "pre-commit": "bash scripts/lint"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
add here #master to tdex-protobuf dependency

Reply:
Done